# x,y,z should be direction of force, in WAM coordinates and units of meters
# f should be force, in Newtons
# Force will be applied at the current endpoint position in the
# specified direction
float64 x
float64 y
float64 z
float64 f
---
bool ok
string reason
uint32 id
